projects/demo/src/modules/app/stackblitz/utils.ts

```diff
@@ -0,0 +1,66 @@
+import {FrontEndExample} from '../../interfaces/front-end-example';
+import {TsFileComponentParser, TsFileModuleParser, TsFileParser} from '../classes';
+import {isLess, isMainComponentFile, isTS} from '../utils';
+
+export const prepareLess = (content: string): string => {
+    return content.replace(
+        '~@taiga-ui/core/styles/taiga-ui-local',
+        '@taiga-ui/core/styles/taiga-ui-local.less',
+    );
+};
+
+export const appPrefix = (stringsPart: TemplateStringsArray, path: string = '') =>
+    `src/app/${stringsPart.join('')}${path}`;
+
+type FileName = string;
+type FileContent = string;
+
+export const getSupportFiles = <T extends FrontEndExample>(
+    files: T,
+): Array<[FileName, FileContent]> => {
+    return Object.entries(files).filter(
+        ([fileName, content]) => content && !isMainComponentFile(fileName),
+    );
+};
+
+export const prepareSupportFiles = (
+    files: Array<[FileName, FileContent]>,
+): Record<FileName, FileContent> => {
+    return files
+        .map(([fileName, fileContent]) => [
+            fileName,
+            isLess(fileName) ? prepareLess(fileContent) : fileContent,
+        ])
+        .reduce(
+            (acc, [fileName, fileContent]) => ({
+                ...acc,
```

Comment:
What do you think about rewriting this code without copying objects?

```ts
export const prepareSupportFiles = (
    files: Array<[FileName, FileContent]>
): Record<FileName, FileContent> => {
    const processedContent: Record<FileName, FileContent> = {};

    for (const file of files) {
       const [fileName, fileContent] = Object.entries(file);
       const prefixedFileName = appPrefix`${fileName}`;
       processedContent[prefixedFileName] = isLess(fileName) ? prepareLess(fileContent) : fileContent
    }

    return processedContent;
}
```

Reply:
done